python/hsfs/engine/python.py

```diff
@@ -872,7 +872,9 @@ def _write_dataframe_kafka(
 
             reset_offsets = (
                 feature_group._online_topic_name
-                not in producer.list_topics(timeout=1).topics.keys()
+                not in producer.list_topics(
+                    timeout=offline_write_options.get("kafka_timeout", 6)
```

Comment:
Shouldn't this property be part of the `kafka_producer_config` dict?

Reply:
Nope that doesn't work, the Producer will compain:
```
KafkaException: KafkaError{code=_INVALID_ARG,val=-186,str="No such configuration property: "kafka_timeout""}
```